### PR TITLE
Add file field for attaching documents to brief response

### PR DESCRIPTION
**Pull request by maxious**

### CODE REVIEW

File: app/main/views/briefs.py [hunk 1]
```diff
@@ -196,6 +198,25 @@ def submit_brief_response(brief_id):
     section = content.get_section(content.get_next_editable_section_id())
     response_data = section.get_data(request.form)
 
+    service = {'frameworkSlug': brief['frameworkSlug'], 'supplierCode': current_user.supplier_code, 'id': brief_id}
+    uploaded_documents, document_errors = upload_service_documents(
+        S3(current_app.config['S3_BUCKET_NAME'], "s3-"+current_app.config['AWS_DEFAULT_REGION']+".amazonaws.com"),
```
Comment (matt-sm) [inline app/main/views/briefs.py]: put the whole region string in config?

File: app/templates/components/upload.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
+{% set error_classes = 'file-upload-input file-upload-input-with-error invalid' if error else 'file-upload-input' %}
```
Comment (matt-sm) [inline app/templates/components/upload.html]: was there already an upload component from the Uk?

Comment (maxious) [inline app/templates/components/upload.html]: yep, this is https://github.com/AusDTO/dto-digitalmarketplace-frontend-toolkit/blob/master/toolkit/templates/forms/upload.html but we refactored it into components at some point

File: app/main/views/briefs.py (hunk 1 above)
Comment (maxious) [inline app/main/views/briefs.py]: AWS_DEFAULT_REGION envvar is part of boto.

**Pull request merged**